source/javascripts/controllers/awards_controller.coffee

```diff
@@ -9,18 +9,32 @@ EGA.controller "AwardsController", ($scope, $http, filterPipeline) ->
   $scope.filters =
     category: names: [], filterMap: {}
     year: names: [], filterMap: {}
+    countries: names: [], selected: []
 
   configurePipeline = ->
     filterPipeline.setCategoryChoices $scope.filters.category.filterMap
     filterPipeline.setYearChoices $scope.filters.year.filterMap
 
+    if anyCountriesSelected()
```

Comment:
I think it should be "any country" instead of "any countries". what does our english native @pferdefleisch suggest?